src/Tests.Microsoft.Security.Utilities.Core/SecretMaskerTests.cs

```diff
@@ -157,13 +157,14 @@ public void SecretMasker_LowConfidenceSecurityModels_Masking()
     private void ValidateSecurityModelsMasking(IEnumerable<RegexPattern> patterns, IRegexEngine engine, bool lowEntropyModels)
 
     {
+        using var assertionScope = new AssertionScope();
```

Comment:
>assertionScope [](http://example.com/codeflow?start=18&length=14)

this actually has no effect. i've been unable to figure out the magic to allow all failures in this test to be accumulated and reported. perhaps this logic would have to be inlined i the upstream caller? putting a pin in it for now